src/main/resources/aws/object-storage.yaml

```diff
@@ -0,0 +1,42 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Creates an S3 bucket for storage of resources (eg. avatars/attachments) outside shared home.
+Parameters:
+  S3StorageBucketName:
+    Type: String
+  AWSAccount:
+    Type: String
+Resources:
+  S3Storage:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName:
+        Ref: S3StorageBucketName
+      LifecycleConfiguration:
```

Comment:
The bucket contents get automatically deleted by AWS in 1 day using a lifecycle configuration.  This reduces the complexity and increases the reliability of deleting the files in the bucket. The alternative of deleting files from the JPT code is both slow and error-prone (ie if JPT stops then the files would never get deleted).

Reply:
Another side effect of this is that we need permission to set the lifecycle configuration for buckets.  This will require adding a couple of permissions to the role JPT runs as.